tls/s2n_connection.h

```diff
@@ -188,3 +209,9 @@ int s2n_connection_kill(struct s2n_connection *conn);
 /* Send/recv a stuffer to/from a connection */
 int s2n_connection_send_stuffer(struct s2n_stuffer *stuffer, struct s2n_connection *conn, uint32_t len);
 int s2n_connection_recv_stuffer(struct s2n_stuffer *stuffer, struct s2n_connection *conn, uint32_t len);
+
+extern int s2n_connection_set_cert_auth_type(struct s2n_connection *conn, s2n_cert_auth_type cert_auth_type);
+extern int s2n_connection_set_verify_cert_chain_cb(struct s2n_connection *conn, verify_cert_trust_chain *callback, void *context);
```

Comment:
I don't think we can add these to the API yet, since there is code in [Part 3/4] that is needed before the API Changes [Part 4/4]